net-mgmt/xymon-client/+POST_INSTALL.post

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# We need to add one "include" line in the default config file from xymon-client
+# to load values from our own template
+
+INCLUDE="include /usr/local/etc/xymon/xymonclient.cfg"
+FILE=/usr/local/www/xymon/client/etc/xymonclient.cfg
```

Comment:
when possible, I would prevent saving settings in the `www` directory, the post install script can be avoided by shipping the `xymonclient.cfg` for this plugin and using the template directory to flush it to disk (probably in stead of  `/usr/local/etc/xymon/xymonclient.cfg` further below)

Reply:
@AdSchellevis like https://github.com/opnsense/plugins/pull/4260/files#r1835329433

I don't think this would be possible. The file `/usr/local/www/xymon/client/etc/xymonclient.cfg` is statically coded from file `/usr/local/www/xymon/client/etc/clientlaunch.cfg`
```
# cat /usr/local/www/xymon/client/etc/clientlaunch.cfg
#
# The clientlaunch.cfg file is loaded by "xymonlaunch".
# It controls which of the Xymon client-side modules to run, 
# (both the main client "xymonclient.sh" and any client-side
# extensions); how often, and with which parameters, options 
# and environment variables.
#
# Note: On the Xymon *server* itself, this file is normally 
#       NOT used. Instead, both the client- and server-tasks
#       are controlled by the tasks.cfg file.
#

# msgcache is used for passive clients, that cannot connect
# directly to the Xymon server. This is not the default
# setup, so this task is normally disabled.
[msgcache]
	DISABLED
	ENVFILE $XYMONCLIENTHOME/etc/xymonclient.cfg
	CMD $XYMONCLIENTHOME/bin/msgcache --no-daemon --pidfile=$XYMONCLIENTLOGS/msgcache.pid
	LOGFILE $XYMONCLIENTLOGS/msgcache.log

# The main client task
[client]
	ENVFILE $XYMONCLIENTHOME/etc/xymonclient.cfg
	CMD $XYMONCLIENTHOME/bin/xymonclient.sh 
	LOGFILE $XYMONCLIENTLOGS/xymonclient.log
	INTERVAL 5m

```
Which is called by the `/usr/local/etc/rc.d/xymon-client` from the non-overridable variable `command_args="--config=/usr/local/www/xymon/client/etc/clientlaunch.cfg --log=/usr/local/www/xymon/client/logs/clientlaunch.log --pidfil
e=${pidfile}"`


Were you suggesting to completely overwrite the `/usr/local/etc/rc.d/xymon-client` file from the package to pass our own arguments?

Reply:
I did some tests to override the `/usr/local/etc/rc.d/xymon-client` and setting `pidfile="/var/run/xymonclient.pid"` (as suggested [here](https://github.com/opnsense/plugins/pull/4260/files#r1797725123)). This is preventing the service to start as `/var/run/` is owned by `root:wheel` and perms are `0755` and the service is started as user `xymon` which don't have permission there.

Reply:
> Next issue is giving execution bit to the /usr/local/etc/rc.d/xymon-client now managed by the template system :(

Looks like I can't set it to executable

https://github.com/opnsense/core/blob/84437b3812d959a7aca7f1af8d6c528d8683607d/src/opnsense/service/modules/template.py#L287-L289

So I don't know how I can accomplish what you requested with this comment, neither how to [change the pidfile path](https://github.com/opnsense/plugins/pull/4260/files#r1797725123)

Reply:
File removed here in favor of ports update see PR opnsense/ports#206